native/src/modules/endpoint/sagas/fetchResourceCache.js

```diff
@@ -19,6 +19,26 @@ const createErrorMessage = (fetchResult: FetchResultType) => {
     `${message}'Failed to download ${result.url} to ${path}': ${result.errorMessage}\n`, '')
 }
 
+function * watchOnProgress (): Saga<void> {
```

Comment:
Please follow the Airbnb Style Guide:   https://github.com/airbnb/javascript#generators--spacing

Reply:
My auto formatter does not know that, perhaps we could teach that eslint?

Either way, done.

Reply:
Our linter does not allow this:
// good
function* foo() {
  // ...
}

I have to use this (to make the linter happy):
// bad
function * foo() {
  // ...
}